PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -1516,7 +1516,7 @@ InitFirmwareRecovery (
     return Status;
   }
 
-  ClearFailedBootCount ();
+  ClearRecoveryTrigger ();
```

Comment:
only enable this only when PcdSblResiliencyEnabled is enabled?

Reply:
This entire function will only be executed when PcdSblResiliencyEnabled is set. Thus, a check here would be redundant.